appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: 1.9.0.{build}
+environment:
+  TOKEN:
+    secure: YOUR-ENCRYPTED-CHOCO-APIKEY

```

Comment:
AppVeyor settings can also be configured in web UI. Perhaps this works too. 

In an exported appveyor.yml you can add encrypted envs which are only set if a build in your own branches is triggered. Others can't get your decrypted env through a PR.